packages/explorer/src/EmbeddedExplorer.ts

```diff
@@ -164,7 +166,14 @@ export class EmbeddedExplorer {
       showHeadersAndEnvVars: displayOptions?.showHeadersAndEnvVars !== false,
       theme: displayOptions?.theme ?? 'dark',
       shouldShowGlobalHeader: true,
-      parentSupportsSubscriptions: false,
+      parentSupportsSubscriptions:
+        !!graphRef &&
+        packageJSON.version &&
+        parseInt(packageJSON.version.replace(/\./, ''), 10) >=
+          parseInt(
+            MIN_VERSION_WITH_SUBSCRIPTIONS_SUPPORTED.replace(/\./, ''),
+            10
+          ),
```

Comment:
we don't need this check here - we can just pass `parentSupportsSubscriptions: !!graphRef`. The reason we had version in studio-ui was because folks could be on old versions of the embeddable explorer package / repo, but we are shipping this code ourselves so anyone who gets this logic will have access to subscriptions 👍